src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -21,8 +26,32 @@ public LineController(final LineService lineService) {
     }
 
     @PostMapping
-    public ResponseEntity createLine(@RequestBody LineRequest lineRequest) {
+    public ResponseEntity<?> createLine(@RequestBody LineRequest lineRequest) {
```

Comment:
LineResponse가 리턴되는 것이 명확한 상황이니 아래와 같이 리턴 타입을 지정하면 어떨까요?

```suggestion
    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
```